### ISSUE TITLE
Crash from calling API 28+ methods on Oreo device

### ISSUE THREAD

**Issue by jsoberg** [lionscribe/notification-channel-compat#1]
### Steps to reproduce problem:

1.  Install example app.
1. Open example app.

### What happened:
App crashes from trying to call API 28 methods on an API 26 device.

Relevant stack trace:
```
2019-07-08 10:31:25.966 22617-22617/com.lionscribe.open.notificationchannelcompat.example E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.lionscribe.open.notificationchannelcompat.example, PID: 22617
    java.lang.NoSuchMethodError: No virtual method setDescription(Ljava/lang/String;)V in class Landroid/app/NotificationChannelGroup; or its super classes (declaration of 'android.app.NotificationChannelGroup' appears in /system/framework/framework.jar)
        at com.lionscribe.open.notificationchannelcompat.NotificationChannelGroupCompat.setDescription(NotificationChannelGroupCompat.java:177)
        at com.lionscribe.open.notificationchannelcompat.example.MainActivity.initChannels(MainActivity.java:84)
        at com.lionscribe.open.notificationchannelcompat.example.MainActivity.onCreate(MainActivity.java:49)
        at android.app.Activity.performCreate(Activity.java:7183)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1220)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2910)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3032)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1696)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6944)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
```
There are a few other places I noticed where this takes place in the code:
- **NotificationChannelGroupCompat, line 164**: (NotificationChannelGroup.getDescription() is API 28)
- **NotificationChannelGroupCompat, lines 214 and 227**: (NotificationChannelGroup.isBlocked() is API 28)
- **NotificationChannelManagerHelper, line 356**: (NotificationManager.getNotificationChannelGroup(channelGroupId) is API 28)

### What should happen:
These calls should be directed elsewhere for Oreo devices, since API 26 doesn't provide these calls. I don't immediately know if there are equivalent calls for API 26.

### Device Information:
- Device: Samsung S7
- Android Revision: 8.0.0
- NotificationChannelCompat Revision: master, e3c6c6

**Comment by lionscribe** [comment]
Thank you for report. Bug has been fixed in version 1.0.2  Bug was due to not relizing that SDK 28 added some Group functionality that was not available in SDK 26

**Comment by jsoberg** [comment]
Thanks!
